scopes/preview/preview/preview.main.runtime.tsx

```diff
@@ -144,7 +151,7 @@ export class PreviewMain {
   }
 
   async getPreview(component: Component): Promise<PreviewArtifact | undefined> {
-    const artifacts = await this.builder.getArtifactsVinylByExtensionAndTaskName(
+    const artifacts = await this.builder.getArtifactsVinylByExtensionAndName(
```

Comment:
Extension and name is different than an extension and task name
name is the name of the artifact while task name is a different prop with a different value.
so this will break that function